src/app/home/home.component.html

```diff
@@ -34,8 +34,8 @@ <h1 class="page-header">EnergyPlus</h1>
         of utilities including IDF-Editor for creating input files using a simple spreadsheet-like interface, EP-Launch
         for managing input and output files and performing batch simulations, and EP-Compare for graphically comparing
         the results of two or more simulations. Several comprehensive
-        <a href="https://www.buildingenergysoftwaretools.com/?capabilities=Whole-building+Energy+Simulation&keywords=EnergyPlus">
-          graphical interfaces</a> for EnergyPlus are also available. DOE does most of its work with EnergyPlus using the
+        <a href="https://www.ibpsa.us/best-directory-list/">
+          tools</a> for EnergyPlus are also available. DOE does most of its work with EnergyPlus using the
```

Comment:
I can't provide a link directly to the GUI search results, so I just called it "tools."  Open to suggestion, but honestly I think it's fine.

Reply:
Yeah, I think that's as good as you can do in this situation..